frontend/src/views/pages/MainPage.tsx

```diff
@@ -1,49 +1,87 @@
-        title="Upload your dataset."
-        stepnum={1}
-        body="The format of the dataset must be a parquet file with the following mandatory columns highlighted in the table below."
-      />
-      <Table />
-      <ProcessStep
-        title="View our feedback."
-        stepnum={2}
-        body="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean molestie mi id sapien posuer."
-      />
-      <ProcessStep
-        title="Adjust and try again!"
-        stepnum={3}
-        body="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean molestie mi id sapien posuer."
-      />
+    <div className="main-page">
+      <Header />
+
+      <Title label="BYE BYE BIAS"/>
 
+      <div className="arrow-container">
+       <a href="#about">
+          <img src={DownArrow} alt="Down Arrow" id="down-arrow" />
+        </a>
+      </div>
+
+      <section id="about">
+        <div className="flex-container">
+          <HeroSection
+            heading={
+              <>
+              Target <span className="highlight">fraud detection</span> bias at the source.
+              </>
+            }
+            body={
+              <>
+              Visualize and alleviate bias in transaction datasets fed 
+              into fraud detection AI models. Check out our <a href="#handy-guide"><span className="highlight"> handy guide ↓</span>
+              </a> if you need any help!
+              </>
+            }
+          />
+          <img src={GraphImage} alt="Graph Image" id="graph-image" />
+        </div>
+      </section>
+      <section id="handy-guide">
+        <div className="process-container">
+          <ProcessStep
+            title={
+              <>
+              <span className="highlight">Upload</span> your dataset.
+              </>
+            }
+            stepnum={1}
+            body="Begin by uploading your dataset, either locally or via an Amazon S3 bucket link.
+            Ensure the dataset is in the Parquet file format and includes the mandatory columns specified in the table below.
+            For tailored analysis, select the protected attributes you wish to analyze."
+          />
+          <Table />
+          <ProcessStep
+            title={
+              <>
+              <span className="highlight">View</span> our feedback.
+              </>
+            }
+            stepnum={2}
+            body="Review the overall fairness score and bias metric results using the provided graphs.
+            Enlarge the graphs for a more detailed view. Access detailed descriptions to understand the rationale
+            behind the scores and gain insights into the metrics."
+          />
+          <ProcessStep
+            title={
+              <>
+              <span className="highlight">Adjust</span> and try again!
+              </>
+            }
+            stepnum={3}
+            body="Modify the dataset based on the bias results. Submit the updated file for analysis to determine
```

Comment:
Instead of "bias results" fairness score? Or just results in general but that is nitpicky